src/main/java/com/revature/backend/model/Analysis_Item.java

```diff
@@ -0,0 +1,116 @@
+package com.revature.backend.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "analysisItems")
+public class Analysis_Item {
```

Comment:
Please change this to PascalCase, not Upper_Snake_Case.

Reply:
@ConnorMRyan see comment from William above regarding `PascalCase`